apps/chat-e2e/src/tests/chatSelectionFunctionality.test.ts

```diff
@@ -0,0 +1,251 @@
+          Colors.backgroundAccentSecondary,
+        );
+        await conversationAssertion.assertEntityDotsMenuState(
+          { name: secondConversation.name },
+          'hidden',
+        );
+      },
+    );
+
+    await dialTest.step('Click on Rename, rename and confirm', async () => {
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.rename);
+      firstConversation.name = 'Renamed chat';
+      await conversations.editConversationNameWithTick(firstConversation.name);
+      await conversations.getEntityByName(firstConversation.name).waitFor();
+      await conversationAssertion.assertSelectedConversation(
+        secondConversation.name,
+      );
+      await conversationAssertion.assertEntityState(
+        { name: firstConversation.name },
+        'visible',
+      );
+    });
+
+    await dialTest.step('Click on Compare', async () => {
+      await conversations.openEntityDropdownMenu(firstConversation.name);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.compare);
+      await conversationAssertion.assertSelectedConversation(
+        firstConversation.name,
+      );
+      await expect
+        .soft(
+          compareConversation.getElementLocator(),
+          ExpectedMessages.conversationToCompareVisible,
+        )
+        .toBeVisible();
+    });
+
+    await dialTest.step('Click on Replay', async () => {
+      replayConversation =
+        ExpectedConstants.replayConversation + secondConversation.name;
+      await conversations.openEntityDropdownMenu(secondConversation.name);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.replay);
+      await conversations.getEntityByName(replayConversation).waitFor();
+      await conversationAssertion.assertSelectedConversation(
+        replayConversation,
+      );
+    });
+
+    await dialTest.step('Click on Playback', async () => {
+      playbackConversation =
+        ExpectedConstants.playbackConversation + firstConversation.name;
+      await conversations.openEntityDropdownMenu(firstConversation.name);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.playback);
+      await conversations.getEntityByName(playbackConversation).waitFor();
+      await conversationAssertion.assertSelectedConversation(
+        playbackConversation,
+      );
+    });
+
+    await dialTest.step('Click on Export', async () => {
+      await conversations.openEntityDropdownMenu(replayConversation);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.export);
+      const downloadedData = await dialHomePage.downloadData(() =>
+        conversations
+          .getDropdownMenu()
+          .selectMenuOption(MenuOptions.withoutAttachments),
+      );
+      expect(downloadedData.path).toBeTruthy();
+      expect(downloadedData.path).toMatch(/\.json$/);
```

Comment:
we have such verification in other tests. 
The goal of current test is to check that exported conversation is selected. I'd leave only assertion below

Reply:
will not be done